index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Pirate Bartender App</title>
+	<meta name="description" content="An app for pirate cocktails">
+	  <!-- Normalize.css, a cross-browser reset file -->
```

Comment:
Consider adding meta charset in head.
http://www.w3schools.com/tags/tag_meta.asp